integration-tests/deployment/ccip/state.go

```diff
@@ -38,6 +40,14 @@ type CCIPChainState struct {
 	RMNRemote          *rmn_remote.RMNRemote
 	// TODO: May need to support older link too
 	LinkToken *burn_mint_erc677.BurnMintERC677
+	// Map between token Descriptor (e.g. LinkSymbol, WethSymbol)
+	// and the respective token contract
+	// This is more of an illustration of how we'll have tokens, and it might need some work later to work properly.
+	BurnMintTokens677 map[TokenSymbol]*burn_mint_erc677.BurnMintERC677
```

Comment:
are these intended to non-fee priced tokens like custom BPS tokens (fee tokens should already be captured with the Weth9 and LinkToken wrappers)? If so they won't necessarily be B/M 677

Reply:
> are these intended to non-fee priced tokens like custom BPS tokens 

This was the intention. What are your suggestions? We can have multiple maps for different kinds as well. For now this is for illustrative purpose. Maybe I can add more comments to explain this.

Reply:
This doesn't seem to be used, any reason we have them here?